js/test/strings-ext/testlistfmt_my_MM.js

```diff
@@ -0,0 +1,253 @@
+/*
+ * testlistfmt_my_MM.js - test the list formatter object
+ *
+ * Copyright © 2020, JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSe-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+if (typeof(ListFmt) === "undefined") {
+    var ListFmt = require("../../lib/ListFmt.js");
+}
+
+if (typeof(ilib) === "undefined") {
+    var ilib = require("../../lib/ilib.js");
+}
+
+module.exports.testlistfmt_my_MM = {
+    setUp: function(callback) {
+        ilib.clearCache();
+        callback();
+    },
+    testListFmtNumberFormatOne_my_MM: function(test) {
+        var fmt = new ListFmt({
+            locale: "my-MM"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["တစ္"]), "တစ္");
+        test.done();
+    },
+    testListFmtNumberFormatTwo_my_MM: function(test) {
+        var fmt = new ListFmt({
+            locale: "my-MM"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["တစ္", "ႎႀစ္"]), 'တစ္နှင့် ႎႀစ္');
+        test.done();
+    },
+    testListFmtNumberFormatThree_my_MM: function(test) {
+        var fmt = new ListFmt({
+            locale: "my-MM"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["တစ္", "ႎႀစ္", "သံုး"]), 'တစ္ ႎႀစ္နှင့် သံုး');
+        test.done();
+    },
+    testListFmtNumberFormatFour_my_MM: function(test) {
+        var fmt = new ListFmt({
+            locale: "my-MM"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["တစ္", "ႎႀစ္", "သံုး", "ေလး"]), 'တစ္ ႎႀစ္ သံုးနှင့် ေလး');
```

Comment:
Not sure if it is github that is rendering these incorrectly, but it looks like a number of combining characters here are not combining with their base characters. Maybe the translations are wrong?

Reply:
I copied the text from https://l-lingo.com/free-lessons/en/learn-burmese/numbers-1-10.html.  because google translate doesn't provide Burmese language. Should I change to another text?

Reply:
It probably doesn't matter. The point is the list format, not the actual words.